content/english/blog/post-oscar-text-to-speech.md

```diff
@@ -12,116 +12,142 @@ draft: false
 ---
 
 
-This use case implements text to speech transformation using the OSCAR serverless platform and the [Google Speech](https://pypi.org/project/google-speech/) library, where an input of plain text returns an audio file.
+This use case implements text-to-speech transformation using the OSCAR serverless platform, where an input of plain text returns an audio file. There are two examples of implementing this use case:
+
+- [Google Speech](https://pypi.org/project/google-speech/) library
+- Pretrained model of [Coqui](https://github.com/coqui-ai/TTS)
 
 ### Previous step: Deploy the OSCAR cluster on an IaaS Cloud and install OSCAR-CLI
 
-Follow the [deployment instructions with IM Dashboard](https://docs.oscar.grycap.net/deploy-im-dashboard/). Or you can execute this script to install it locally.
+Follow the [deployment instructions with IM Dashboard](https://docs.oscar.grycap.net/deploy-im-dashboard/). Alternatively, can be executed this script to install locally.
 
 ```bash
 curl -sSL http://go.oscar.grycap.net | bash
 ```
-To create the function, we will use the command line interface [OSCAR-CLI](https://docs.oscar.grycap.net/oscar-cli/).
+
+To create the function, we will use the command-line interface [OSCAR-CLI](https://docs.oscar.grycap.net/oscar-cli/).
 
 ![01-oscar-login.png](../../images/blog/post-text-to-speech/01-oscar-login.png)
 
+### STEP 1: Deploy the Service
 
+We will use OSCAR-CLI to deploy the use-case service. You can see all files of [Google speech](https://github.com/orgs/grycap/packages/container/package/text-to-speech-google) and [Coqui](https://github.com/orgs/grycap/packages/container/package/text-to-speech-coqui) examples in Github.
```

Comment:
These links point to GCR. Instead, they should point to their corresponding repositories in https://github.com/grycap/oscar/tree/master/examples